### ISSUE TITLE
OpenShot 2.2 crashing when I change time.

### ISSUE THREAD

**Issue by DawidcheasePL** [OpenShot/openshot-qt#496]
When I change time OpenShot is crashing

**Comment by Pebin** [comment]
I would add that it crashes for me both on Windows 10 and OpenSUSe 42.1 so it's probably not platform dependent. I even tried to run it from source but it crashes too.

**Comment by thenotec** [comment]
More crashes derive in using dark theme. Try to use No theme in settings

**Comment by astaaah** [comment]
I'm having the same issue :( Does anyone know how to make it work?

**Comment by JtLively** [comment]
Does it crash when you change time as in jumping around on the timeline? If so, does it still crash if you _stop_ before moving the timeline?

**Comment by astaaah** [comment]
Jt: it crashes when I change the time, as in try to speed up/slow down a clip...

**Comment by jonoomph** [comment]
Can we verify this bug still happens on our latest daily build? Please visit www.openshot.org/download, click '**Other Downloads**', and install the latest daily build of OpenShot. If so, please re-open this issue, and attach log files of the crash (https://github.com/OpenShot/openshot-qt/blob/master/.github/CONTRIBUTING.md), and we'll get it fixed. =) Thanks.

**Comment by Pebin** [comment]
It still crashes. At least for me. I tried this build OpenShot-v2.2.0-36-g1a4e1d0-49-608-x86_64 on Win 10

Steps to trigger the crash are simple. Add video, add it to timeline, right click the track, select time and try to speed up the video. Then as soon as it starts playing it will crash.

I checked the logs but I feel like I was getting little bit different exception last time. Some kind of memory access issue. (But maybe I was on my linux machine then. I can checkout the latest source code on linux and try to run that for logs if you want.) .. 

[libopenshot.log.txt](https://github.com/OpenShot/openshot-qt/files/819568/libopenshot.log.txt)
[openshot-qt.log.txt](https://github.com/OpenShot/openshot-qt/files/819569/openshot-qt.log.txt)

I don't know how to reopen issue or if I can do that at all :(
